src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/test_storage.py

```diff
@@ -93,7 +93,7 @@ def body(self):
         s.cmd('storage account delete -g {} -n {} --force'.format(rg, account))
         s.cmd('storage account check-name --name {}'.format(account), checks=JMESPathCheck('nameAvailable', True))
         result = s.cmd('storage account check-name --name teststorageomega --query "nameAvailable" -o tsv')
-        assert result == 'false'
+        assert result == 'true'
```

Comment:
In the same scenario, the account name is checked and ensure to be available at the beginning. There isn't any operations against the 'teststorageomega', therefore I think this change make sense here: it should keep be available.